giver/templates/giver/overlay.html

```diff
@@ -3,25 +3,97 @@
 <html>
   <head>
     <script src="https://app.ohmnilabs.com/api/Ohmni.js"></script>
-    <script>
-     var showFace = function(event) {
-       Ohmni.showPageOnBot("https://www.carebyfern.com/face");
-       console.log('faced')
-     }
-     var hideFace = function(event) {
-       Ohmni.hidePageOnBot();
-       console.log('unfaced')
-     }
-     document.addEventListener("DOMContentLoaded", showFace)
-    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+
+
   </head>
   <body>
+    
+   
+    <div>
+      <button type="button" data-external="true" class="button" style="padding:30px;margin:20px;" onclick="hideFace()">Hide face</button>
+    </div>
+    
+    
     <div style="margin-bottom:1em">
-      <button type="button" onclick="showFace()">Show face</button>
+      <button type="button" data-external="true" class="button" style="padding:30px;margin:20px" onclick="showFace()">Show face</button>
     </div>
+
     <div>
-      <button type="button" onclick="hideFace()">Hide face</button>
+      <text> Current URL: </text><text id="current-url">None</text>
+      <br>
+      <input id="target-url" type="text" text="URL"/>
+      <button type="button" id="url-submit" onclick="setUrl()">Go to URL</button>
     </div>
+    <!--
+    <div>
+      <button id="face-toggle" type="button" data-external="true" class="button" style="padding:30px;margin:20px;font-weight:bold" onclick="toggleFace()">Show Face</button>
+      <text style="color:red">FACE TOGGLE</text>
+    </div>
+    -->
 
   </body>
+      <script>
+    var tries = 0;
+     var showFace = function(event) {
```

Comment:
set tries back to 0?

Reply:
Shouldn't set tries back to zero because I think it works ok once the page is loaded the first time - otherwise we should try multiple tries on any url load.